akka-actor/src/main/scala/akka/pattern/CircuitBreaker.scala

```diff
@@ -983,7 +1001,8 @@ class CircuitBreaker(
       scheduler.scheduleOnce(currentResetTimeout) {
         attemptReset()
       }
-      val nextResetTimeout = currentResetTimeout * exponentialBackoffFactor match {
+      val rnd = 1.0 + ThreadLocalRandom.current().nextDouble() * randomFactor
```

Comment:
(TIL `nextDouble` returns a double between 0.0 and 1.0)

So the random factor can increase the reset timeout but not decrease it? OK I guess, and consistent with the delay used in the RestartSupervisor - but as said important to document.

Reply:
The range is documented in scaladoc, thanks for point that out